3.evaluate_model/scripts/nbconverted/get_LOIO_probabilities.py

```diff
@@ -0,0 +1,168 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# In[1]:
+
+
+import pathlib
+import warnings
+
+import pandas as pd
+
+from sklearn.linear_model import LogisticRegression
+from sklearn.utils import parallel_backend
+from sklearn.exceptions import ConvergenceWarning
+from joblib import load
+
+import sys
+
+sys.path.append("../utils")
+from split_utils import get_features_data
+from train_utils import get_dataset, get_X_y_data
```

Comment:
Was `get_dataset` used? If not, consider removing.

Reply:
Removed in https://github.com/WayScience/phenotypic_profiling_model/pull/23/commits/0ca85f5f0e452c6651eeb2d8d24464bb24058b62